STYLEGUIDE.md

```diff
@@ -43,7 +44,7 @@ Assume the reader is a busy non-native English speaker.
       - Sourcegraph['s] Firefox add-on
       - Sourcegraph['s] Safari extension
 
-When specifically distinguishing between Server and Data Center, it might help to say "the [single-node] Server deployment option" and "the Data Center [cluster] deployment option".
+When specifically distinguishing between Core and OSS it is important to note that Core is not built from the OSS code base and OSS does not include tracking data, ability to upgrade, or Sourcegraph extensions. These are included in the Core version of Sourcegraph, that is built from the same code as Sourcegraph Enterprise and has the ability to upgrade later.
```

Comment:
> does not include tracking data
We don't collect any telemetry from "dev"/oss, but we also haven't ripped this code out of the common codebase. I'd remove this for now.